libmscore/note.cpp

```diff
@@ -3429,7 +3429,7 @@ Shape Note::shape() const
       Shape shape(r, name());
       for (NoteDot* dot : _dots)
             shape.add(symBbox(SymId::augmentationDot).translated(dot->pos()), dot->name());
-      if (_accidental)
+      if (_accidental && _accidental->addToSkyline())
```

Comment:
For the purpose of making sure accidentals can tuck over/under other notes, this is not needed.  Which is to say, it could be visible() instead of addToSkyline().  As it is, this change means non-autoplaced accidentals can not only tuck over/under, they can also collide with previous notes.  I guess that's expected, although not what I was originally thinking might be nice.  I suppose it's more of a hack, though, so I'm fine leaving this as you have it.  Just wanted to mention it in case it sparks any additional thoughts.